bots/tests-scan

```diff
@@ -80,10 +80,23 @@ def main():
                         help='Don''t actually change anything on GitHub')
     parser.add_argument('--repo', default=None,
                         help='Repository to scan and checkout.')
+    parser.add_argument('--module-repo', default=None,
+                        help='Repository to scan and incorporate into Cockpit.')
+    parser.add_argument('--module-updater', default=None,
+                        help='Program to call to update Cockpit with the module.')
     parser.add_argument('-c', '--context', action="append", default=[ ],
                         help='Test contexts to use.')
     opts = parser.parse_args()
-    api = github.GitHub(repo=opts.repo)
+
+    if opts.repo and opts.module_repo:
+        sys.stderr.write("tests-scan: Only one of --repo and --module-repo can be specified.\n")
+        return 1
```

Comment:
I suggest using `parser.error("blah")`. This shows the short help, exits the program with 1, and should generally be used when encountering wrong CLI options.